misc-bench/cpp-batched-gemv/Makefile

```diff
@@ -0,0 +1,16 @@
+# Makefile: C++ version of matrix matrix multiplication
```

Comment:
I believe the header should be batch multiplication of matrix and vector. Might need to be updated for other files as well.